### PR TITLE
Documentation - quickstart, tutorial, dev instructions

### PR DESCRIPTION
**Pull request by shanalily**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,6 +85,16 @@ CONTROLLER_GEN=$(shell which controller-gen)
 endif
 .PHONY: controller-gen
 
-are-you-sure:
-	@echo "Are you sure? [Y/N] " && read ans && [ $${ans:-N} = Y ]
-PHONY: are-you-sure
+quickstart:
```
Comment (juan-lee) [inline Makefile]: A little easier way to do this is to use `envsubst`. You can put ${AZURE_IDENTITY} in the yaml file and then do `cat quickkstart.yaml | envsubst`. `envsubs` will replace those strings with what is set in your environment.

File: README.md [hunk 2]
```diff
@@ -13,53 +13,49 @@ The purpose of this Kubernetes controller is to sync ARM VM/VMSS tags and node l
...
+az role assignment create --role "Managed Identity Operator" --assignee $AZURE_IDENTITY_PRINCIPAL_ID --scope $AZURE_IDENTITY_RESOURCE_ID
+az role assignment create --role "Contributor" --assignee $AZURE_IDENTITY_PRINCIPAL_ID --scope /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}
+```
+
+4. Create k8s resources, including controller deployment. 
+
+Clone this repository. From the root directory of the repository, run:
+
+```sh
+make quickstart
```
Comment (juan-lee) [inline README.md]: If you use `envsubst` you can just say `cat  config/quickstart/quickstart.yaml | envsubst | kubectl apply -f -` here.

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+```sh
+az role assignment create --role "Managed Identity Operator" --assignee $AZURE_IDENTITY_PRINCIPAL_ID --scope $AZURE_IDENTITY_RESOURCE_ID
+az role assignment create --role "Contributor" --assignee $PRINCIPAL_ID --scope /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}
+```
+
+4. Create k8s resources, including controller deployment. 
+
+```sh
+make quickstart
```
Comment (juan-lee) [inline docs/quickstart.md]: don't forget to update this one

File: Makefile (hunk 1 above)
Comment (shanalily) [reply Makefile]: Oh yeah, even if I'd stuck with 'sed' I forgot to clean that up more.

File: .gitignore [hunk 4]
```diff
@@ -23,3 +23,4 @@ tests/aks-engine/aadpodidentity.yaml
 tests/aks-engine/aadpodidentitybinding.yaml
 tests/aks/aadpodidentity.yaml
 tests/aks/aadpodidentitybinding.yaml
+config/quickstart/quickstart.yaml
```
Comment (juan-lee) [inline .gitignore]: can this be removed now?

File: README.md (hunk 2 above)
Comment (shanalily) [reply README.md]: I like that it's easier to see what's happening that way.

File: docs/quickstart.md (hunk 3 above)
Comment (shanalily) [reply docs/quickstart.md]: I ended up deleting that file since it's all in the README.

File: .gitignore (hunk 4 above)
Comment (shanalily) [reply .gitignore]: I almost forgot to do that, thanks.

**Pull request merged**